scripts/AIGame.cpp

```diff
@@ -43,15 +46,15 @@ DLLEXPORT int STDCALL SaveRestoreState(void *file_ptr, uint8_t saving_state);
 #endif
 
 static int String_table_size = 0;
```

Comment:
String_table_size can be removed, it's only assigned once to and not used anywhere else.